### PR TITLE
[Renesas]Add tft-lcd lvgl support

### PR DESCRIPTION
**Pull request by Rbb666**
## 拉取/合并请求描述：(PR description)

[
Add spi screen (ili9341) driver and adapt to lvgl
]

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [x] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [x] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [x] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [x] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [x] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [x] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [x] 代码是高质量的 Code in this PR is of high quality
- [x] 已经使用[formatting](https://github.com/mysterywolf/formatting)等源码格式化工具确保格式符合[RT-Thread代码规范](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_cn.md) This PR complies with [RT-Thread code specification](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_en.md)

### CODE REVIEW

**Comment by Rbb666** [comment]
build

File: bsp/renesas/ra6m3-ek/board/ports/ili9341/lcd_ili9341.c [hunk 1]
```diff
@@ -0,0 +1,334 @@
+#include <rtdevice.h>
+
+#ifdef BSP_USING_SPI_LCD
+#include "lcd_ili9341.h"
+#include "drivers/spi.h"
+
+// 2.8 inch LCD module
```
Comment (Guozhanxin) [inline bsp/renesas/ra6m3-ek/board/ports/ili9341/lcd_ili9341.c:7]: 代码注释，要求 /* 这种形式 */

File: bsp/renesas/ra6m3-ek/board/ports/ili9341/lcd_ili9341.c [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+{
+    lcddev.setxcmd = 0x2A;
+    lcddev.setycmd = 0x2B;
+    lcddev.wramcmd = 0x2C;
+    switch (direction)
+    {
+    case 0:
+        lcddev.width = LCD_W;
+        lcddev.height = LCD_H;
+        LCD_WriteReg(0x36, (1 << 3) | (0 << 6) | (0 << 7)); // BGR==1,MY==0,MX==0,MV==0
```
Comment (Guozhanxin) [inline bsp/renesas/ra6m3-ek/board/ports/ili9341/lcd_ili9341.c:112]: 这些注释也要改

Comment (Rbb666) [reply bsp/renesas/ra6m3-ek/board/ports/ili9341/lcd_ili9341.c]: 改完了

**Review by Guozhanxin — APPROVED** [review]

**Review by mysterywolf — APPROVED** [review]

**Pull request merged**